include/ofi_proto.h

```diff
@@ -49,7 +49,7 @@ extern "C" {
 #define OFI_CTRL_VERSION	2
 
 /* ofi_ctrl_hdr::type */
-enum {
+enum ofi_ctrl_type {
```

Comment:
The main reason this was not named is the values are used in the protocol headers, which are a fixed size (u8).  So a named enum value isn't useful in nearly all places where the values are used.  I would drop this patch

Reply:
Ok fine.